### PR TITLE
Add how to config limit for datadog metrics missing

### PR DESCRIPTION
**Pull request by Aiqin-Aiven**
Add how to config limit for datadog metrics missing

### CODE REVIEW

File: docs/platform/howto/integrations/missing-datadog-metrics.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.. note:: Datadog may also limit the number of custom metrics that can be received, based on your pricing plan_ . 
+
+Steps to increase the limit
+---------------------------
+The ``max_jmx_metrics`` is not exposed in our console yet, but you should be able to change it with the Aiven command line client_, using the following procedure:
+
+1. Find the SERVICE_INTEGRATION_ID for your Datadog integration with
+
+    ``avn service integration-list --project={PROJECT_NAME}``
```
Comment (lornajane) [inline docs/platform/howto/integrations/missing-datadog-metrics.rst:31]: This command should include the name of the service whose metrics we want as the final argument

File: docs/platform/howto/integrations/missing-datadog-metrics.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+We allow setting the maximum metric limit for services where Datadog agent gathers the metrics via ``JMX`` using the ``max_jmx_metrics`` option in the Datadog integration configuration. This can be set to any value between 10 and 100000 with a default value: 2000.
+
+.. note:: Datadog may also limit the number of custom metrics that can be received, based on your pricing plan_ . 
+
+Steps to increase the limit
+---------------------------
+The ``max_jmx_metrics`` is not exposed in our console yet, but you should be able to change it with the Aiven command line client_, using the following procedure:
+
+1. Find the SERVICE_INTEGRATION_ID for your Datadog integration with
```
Comment (lornajane) [inline docs/platform/howto/integrations/missing-datadog-metrics.rst:29]: Use `code/literal` markup for the placeholders such as `SERVICE_INTEGRATION_ID`

**Review by lornajane — DISMISSED** [review]
Thanks @Aiqin-Aiven this looks so good! I added some requests for changes, but all small things. Could you please also link TO this new post from the general article about datadog configuration? And anywhere else that you think users might need this information from.

One question I do have is: how can I find which configuration options are available for a particular service integration type? I wasn't sure how to actively test this without a datadog account - but the first step does work and I am sure the second one is fine, so this is a general question.

File: docs/platform/howto/integrations/missing-datadog-metrics.rst (hunk 1 above)
Comment (Aiqin-Aiven) [reply docs/platform/howto/integrations/missing-datadog-metrics.rst:31]: Oh, yes, missing a service_name there

**Comment by Aiqin-Aiven** [comment]
> ntegration type? I wasn't sure how to actively test this without a datadog account - but the first step does work and I am sure the second one is fine, so this is a general question.

That datadog article is still in intercom? I will have a look and link this there.
For testing, datadog account can be applied freely from datadog website. most of the attribute should be available in console, but this one not yet. there is JIRA ticket towards it.

**Review by lornajane — APPROVED** [review]
Renamed the file and added a navigation link, but otherwise it was in perfect shape. Thanks @Aiqin-Aiven !!

**Pull request merged**